.github/workflows/workflow-test-in-container.yml

```
@@ -0,0 +1,105 @@
+jobs:
+  check:
+    timeout-minutes: 30
+    permissions: {}
+    runs-on: ${{ inputs.runs-on }}
+    container:
+      image: mcr.microsoft.com/playwright:v1.50.1-noble
+      credentials:
+        username: lynx-family
+        password: ${{ secrets.GITHUB_TOKEN }}
+      env:
+        ACCESS_KEY: ${{ secrets.TOS_AP_ACCESS_KEY }}
```

actionlint (1.7.7)

property "tos_ap_access_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; codecov_token: string; github_token: string; lhci_github_app_token: string}

(expression)